src/ports/postgres/modules/utilities/utilities.sql_in

```diff
@@ -541,3 +541,49 @@ BEGIN
     RETURN 0;
 END;
 $$ LANGUAGE plpgsql;
+
+CREATE OR REPLACE FUNCTION MADLIB_SCHEMA.is_ver_greater_than_gp_640_or_pg_11()
```

Comment:
I understand why we need this very specific function but it would be great to add a few lines of comments for context. It is especially confusing because the function is in utilities and I usually expect them to be somewhat generic. Please note that I am not suggesting to move the function to DL, just some comments.

Reply:
makes sense. will do